coreservices/partsrelationshipservice/terraform/outputs.tf

```
@@ -0,0 +1,47 @@
+output "catenax_eventhubs_namespace_name" {
+  value = module.eventhubs_namespace.name
+}
+
+output "catenax_events_topic" {
+  value       = module.eventhub_catenax_events.eventhub_name
+  description = "The Event Hubs name (Kafka topic) for Catena-X events."
+}
+
+output "catenax_events_send_primary_connection_string" {
+  value       = module.eventhub_catenax_events.send_primary_connection_string
+  description = "The primary connection string to send Catena-X events."
+  sensitive   = true
+}
+
+output "catenax_events_listen_primary_connection_string" {
+  value       = module.eventhub_catenax_events.listen_primary_connection_string
+  description = "The primary connection string to receive Catena-X events."
```

nit:
```suggestion
  description = "The primary connection string to listen Catena-X events."
```
up to you to decide.